.evergreen/.evg.yml

```diff
@@ -561,6 +561,23 @@ axes:
         variables:
            JDK: "jdk6"
 
+  # Choice of MongoDB storage engine
+  - id: storage-engine
+    display_name: Storage
+    values:
+      - id: mmapv1
+        display_name: MMAPv1
+        variables:
+           STORAGE_ENGINE: "mmapv1"
+      - id: wiredtiger
```

Comment:
I think you're going to have to add storage-engine=wiretiger to all the other build variants, in order to avoid exploding the test matrix.

Reply:
Did you confirm it was actually necessary to do that?

Reply:
I do not believe it is necessary to do this, so I have removed it for now. I would like to test with all the Evergreen changes to confirm.

Reply:
When you've confirmed, you can resolve this.

Reply:
After testing on Evergreen, I determined it was not necessary to add `storage-engine=wiredtiger` to all the other build variants.